HLTrigger/Configuration/python/customizeHLTforAlpaka.py

```diff
@@ -249,6 +243,21 @@ def replaceItemsInSequence(process, itemsToReplace, replacingSequence):
     itemsList = [process.hltParticleFlowRecHitHBHE, process.hltParticleFlowClusterHBHE,process.hltParticleFlowClusterHCAL]
     process = replaceItemsInSequence(process, itemsList, process.HLTPFHcalClustering)
 
+    process.HLTPFClusterHBHECPUSerial = cms.Sequence(process.hltHBHERecHitToSoA+process.hltPFRecHitSoAProducerHCALCPUSerial+process.hltPFClusterSoAProducerCPUSerial)
+
+    # Add CPUSerial sequences to DQM_HcalReconstruction_v6 Path
+    dqmHcalRecoPathName = "DQM_HcalReconstruction_v6"
+    dqmHcalPath= getattr(process, dqmHcalRecoPathName)
```

Comment:
Seems like this `getattr()` call is causing workflow failures in GPU IB, see https://github.com/cms-sw/cmssw/issues/44046

Reply:
I guess something like this 

https://github.com/cms-sw/cmssw/blob/8f7db2fe757124e04b3fe24a7a1af522dedb47b2/HLTrigger/Configuration/python/customizeHLTforAlpaka.py#L268-L275

is needed. 
@waredjeb FYI

Reply:
thanks for providing the fix with https://github.com/cms-sw/cmssw/pull/44048
should I update #44000 with the same fix?

Reply:
> should I update https://github.com/cms-sw/cmssw/pull/44000 with the same fix?

yes, please